PRODUCTIONISATION.md

```diff
@@ -107,32 +106,38 @@ To be applied if infrastructure has been orchestrated and provisioned specifical
 * [ ] 8.1 Sub-organisation created under parent organisation for each customer
 * [ ] 8.2 Monitoring agent be installed on all production infrastructure
 * [ ] 8.3 Monitoring agent could be installed on non-production infrastructure if necessary
-* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008]
+* [ ] 8.4 Monitoring agent hostname configured according to proposed [RFC 008 - Server naming convention][rfc_008] (_this doesn't apply to serverless_)
 * 8.5 Monitors configured in infrastructure monitoring service for (warn/alert):
   * [ ] 8.5.1 Inode usage (80% / 90%)
   * [ ] 8.5.2 CPU load (75% / 85% average over 2 minutes)
   * [ ] 8.5.3 Disk usage (80% / 90%)
-  * [ ] 8.5.4 Memory usage (80% / 90% average over 2 minutes)
+  * [ ] 8.5.4 Memory usage (60% / 70% average over 2 minutes)
```

Comment:
These figures make for a good rule of thumb, however we should set out some contextual figures at some point.

For instance, if you're hosting an ecs agent cluster, you should ensure that your average CPU usage does not go over `n-1/n`, where `n` is the number of servers, to account for increased load in the event a server goes down.
That is to say, if you have 2 servers (`A` and `B`), then you'd expect neither of them to go above 50% (`1/2`). In the event that `A` goes down, `B` can then pick up the slack until `A` is restored.
The same goes for 3 servers. `2/3 =~ 66%`. if `A` goes down, `B` and `C` pick up `~33%` each.

This should apply for most of the other stats too.

Reply:
Could review this later as well. Created a Trello comment and marking as resolved for now.